core/src/avm2/globals/flash/xml/XMLDocument.as

```diff
@@ -1,6 +1,21 @@
 package flash.xml
 {
    public class XMLDocument extends XMLNode {
```

Comment:
I'd add a comment explaining that this is a quick shim and that eventually we'd want to use a proper legacy XML implementation.